eo-maven-plugin/src/main/java/org/eolang/maven/footprint/FtDefault.java

```diff
@@ -70,4 +74,18 @@ public void save(final String program, final String ext, final Scalar<String> co
             this.main.relativize(new Place(program).make(this.main, ext))
         );
     }
+
+    @Override
+    public List<Path> list(final String ext) throws IOException {
```

Comment:
@volodya-lombrozo I believe we need to cover it with tests.

Reply:
@mximp Done